yt/visualization/profile_plotter.py

```diff
@@ -501,6 +505,46 @@ def set_log(self, field, log):
                 raise KeyError("Field %s not in profile plot!" % (field))
         return self
 
+
+    @invalidate_plot
+    def set_ylabel(self, field, label):
+        """
+
+        """
+        if field == "all":
+            for field in list(self.profiles[0].field_data.keys()):
+                print(field)
+                self.y_title[field] = label
+        else:
+            field, = self.profiles[0].data_source._determine_fields([field])
+            if field in self.profiles[0].field_data:
+                self.y_title[field] = label
+            else:
+                raise KeyError("Field %s not in profile plot!" % (field))
+
+        _plot_valid = False
+
+        return self
+
+    @invalidate_plot
+    def set_xlabel(self, field, label):
```

Comment:
I missed this when I was reviewing yesterday, but the signature of this function should be `set_xlabel(self, label)` since a `ProfilePlot` only ever has one x field.

You also don't need to keep track of the x_title for every field in the implementation of this function below since all of the fields can share the same x title.

Reply:
Yeah, I noticed that. I did it like that so that the xlabel can be changed on specific plots only, if that is desired. But I see how it can be confusing. I don't see why you'd want to use a different xlabel on different plots since it is the same field, just thought it may be desirable

Reply:
Fair enough! Leaving it this way is fine as long as you make it clear in the docstring and the documentation what is going on.

Reply:
@ngoldbaum  I could reverse the ordering of the arguments (label, field) and have it default to all? I'm very new the yt code base and open source, apologies for all the questions.

Reply:
To be honest I'm not sure what the best way to go is. I kinda think it would be best to keep things simple and only allow setting one x label and not having `field` as an argument. However if you think it would be useful to set different xlabels for different y fields the current implementation in this PR is fine and there's no need to change the ordering of the arguments (that would be confusing since the signature of `set_xlabel` and `set_ylabel` would be reversed).

Reply:
Thanks @ngoldbaum, I'll keep things simple then! I'll do the docs and stuff now and submit the updated code later.